index.js

```diff
@@ -410,17 +422,18 @@ module.exports = class Filter extends Plugin {
     }
   }
 
-  async processFile(srcDir, destDir, relativePath, forceInvalidation, isChange, instrumentation, entry) {
+  async processFile(srcDir, relativePath, forceInvalidation, isChange, instrumentation, entry) {
```

Comment:
Changing this signature by dropping `destDir` is a breaking change, as `processFile` is public API for plugin subclasses. I don't think we want to do that, otherwise upgrades will be extra complex for little benefit.